components/RichText/Quote.tsx

```diff
@@ -3,34 +3,36 @@ import { Alignment } from '@prezly/slate-types';
 import classNames from 'classnames';
 import type { PropsWithChildren } from 'react';
 
-import { IconQuoteClosing, IconQuoteOpening } from '@/icons';
-
 import styles from './Quote.module.scss';
 
 interface Props {
     node: QuoteNode;
 }
 
 export function Quote({ node, children }: PropsWithChildren<Props>) {
+    const alignment = node.align ?? Alignment.LEFT;
+
     return (
-        <blockquote className={styles.container}>
-            <div className={styles.left} aria-hidden="true">
-                <IconQuoteOpening className={styles.quote} />
-                <IconQuoteOpening className={styles.quote} />
-            </div>
+        <blockquote
```

Comment:
✖️
Same comment here as in Marcel: it can be made much simpler.

Since this block also has a background, a simple border probably won't work, but you can add it on the inner `div` with the text and achieve the same result.